package.json

```diff
@@ -116,7 +116,7 @@
     "babel-loader": "8.0.5",
     "babel-plugin-dynamic-import-node": "2.2.0",
     "babel-plugin-macros": "2.8.0",
-    "bulma": "0.7.4",
+    "bulma": "0.9.0>",
```

Comment:
Is this a requirement for your fix to work?  If not, I think we should maybe leave it for a separate PR--one where we can make sure that none of the site layout changes when we upgrade!